cfe_internal/enterprise/mission_portal.cf

```diff
@@ -15,6 +15,11 @@ bundle agent cfe_internal_enterprise_mission_portal
         handle => "cfe_internal_management_web_server",
         comment => "Manage Apache Web server (on/off)";
 
+      "Port Configuration" -> { "CFEngine Enterprise", "Mission Portal" }
```

Comment:
Does a configuration change here necessitate a service restart for them to become active?

Even if it does not, I think that it would make sense that this promise be moved in front of the service management to enable the web-server.

Reply:
yes it's needed and it does it.

if you mean this one
>comment => "Manage Apache Web server (on/off)";

this only switches  on/off webserver when a specific class defined